src/solvers/dg.jl

```diff
@@ -238,7 +277,7 @@ function Base.show(io::IO, mime::MIME"text/plain",
     end
 end
 
-function get_node_variables!(node_variables, mesh, equations,
+function get_node_variables!(node_variables, u_ode, mesh, equations,
                              volume_integral::VolumeIntegralSubcellLimiting, dg, cache)
```

Comment:
For VolumeIntegralSubcellLimiting, isn't this function called every time and your new function above never?

Reply:
Ah yeah right, good catch! This is meant to be deleted.